src/hb-ot-layout-gsubgpos.hh

```diff
@@ -3405,6 +3418,42 @@ struct GSUBGPOS
           && !alternate_feature_indices.has (i))
         feature_indices->del (i);
     }
+
+    //find out duplicate features after subset
```

Comment:
Finding duplicate features is a pretty distinct operation from the pruning. So I would make this it's own method, like:
```
 find_duplicate_features ( const hb_map_t *lookup_indices,
                                            const hb_set_t *feature_indices,
                                            hb_map_t *duplicate_index_map /* OUT */)
```